src/graphql/models/User.js

```diff
@@ -24,31 +30,32 @@ export const currentUserOnlyField = (type, resolver) => ({
   },
 });
 
-/**
- * Use Gravatar as avatar provider.
- */
-const avatarResolver = (s = 80, d = 'identicon', r = 'g') => ({
-  type: GraphQLString,
-  description: 'return hash based on user email for gravatar url',
-  resolve(user) {
-    const GRAVATAR_URL = 'https://www.gravatar.com/avatar/';
-    const hash = crypto
-      .createHash('md5')
-      .update((user.email || user.id).trim().toLocaleLowerCase())
-      .digest('hex');
-    const params = `?s=${s}&d=${d}&r=${r}`;
-    return `${GRAVATAR_URL}${hash}${params}`;
-  },
-});
-
 const User = new GraphQLObjectType({
   name: 'User',
   fields: () => ({
     id: { type: GraphQLString },
+    slug: { type: GraphQLString },
     email: currentUserOnlyField(GraphQLString),
     name: { type: GraphQLString },
-    avatarUrl: avatarResolver(),
+    bio: { type: GraphQLString },
+
+    avatarUrl: {
+      type: GraphQLString,
+      description: 'returns avatar url from facebook, github or gravatar',
+      resolve: avatarUrlResolver(),
+    },
     avatarData: { type: GraphQLString },
```

Comment:
nit: suggest adding description on `avatarData`, stating that it should be used only when `avatarType ` is `OpenPeeps`